integration-libs/cdc/root/guards/cdc-logout.guard.spec.ts

```diff
@@ -18,6 +18,10 @@ class MockAuthService implements Partial<AuthService> {
   coreLogout() {
     return Promise.resolve();
   }
+
+  isUserLoggedIn(): Observable<boolean> {
+    return of(true);
+  }
```

Comment:
This function does not seem used in the test.  Can it be removed? If some intended test coverage is missing instead, could we add the coverage?